### DISCUSSION TITLE
Add inline error handling to all member forms with access to errors in the templates

### DISCUSSION THREAD

**Discussion by pyrobob83** [ExpressionEngine/ExpressionEngine#2871] [Feature Requests]
Currently it appears only the member-related form that has inline error handling available is the Registration form - is there a reason for that?  My request is for at least all of these forms (if not all member-related forms) to also have that feature available, with access to the errors in the templates so we can avoid the default EE grey error screens:

- exp:member:login_form (eg, login unsuccessful)
- exp:member:reset_password_form (eg, if the passwords don't match or don't meet security policy)
- exp:member:edit_profile (eg, if username isn't available or new password don't match or meet security policy)

**Comment by intoeetive** [comment]
@pyrobob83 I guess the only reason is that registration form was evolving differently. The template tag existed for a while, while for other forms we were making the existing code for legacy member templates work in EE "regular" templates - which did not support inline errors

This FR completely makes sense (and hopefully implementing inline error would not require major rewrite, LOL)

**Comment by intoeetive** [comment]
https://github.com/ExpressionEngine/ExpressionEngine/pull/4093
